### DISCUSSION TITLE
Verify server certificate against CA

### DISCUSSION THREAD

**Discussion by rehusp** [FreeOpcUa/opcua-asyncio#1317] [Q&A]
Hello everyone,
Is there any way how to verify server certificate against CA (see specification [here](https://reference.opcfoundation.org/v104/Core/docs/Part4/6.1.3/))?
I can see there is only support to specify client (key+cert) and server key ([opcua.client.client.Client.set_security_string](https://python-opcua.readthedocs.io/en/latest/client.html#opcua.client.client.Client.set_security_string)) but I cannot find nothing regarding verifying server certificate against Certification Authority nor Revocation List.

Thanks in advance.

**Comment by schroeder-** [comment]
This feature is currently not implemented.
